docs/advanced-ai/intro-tutorial.md

```diff
@@ -1,217 +1,185 @@
+	For more about credentials
+
+
+## 6. Test the node
+
+Now the node is connected to the **Chat Trigger** and a chat model, we can test this part of the workflow
+
+1. Click on the 'Chat' button near the bottom of the canvas. This opens up a local chat window on the left and the AI agent logs on the right.
+
+1. Type in a message and press ++enter++. You will now see the response from the chat model appear below your message.
+
+1. The log window displays the inputs and outputs to and from the AI Agent.
+   ![image showing a chat session in progress](/_images/advanced-ai/ai-intro-chat.png)
+
+??? explanation "Accessing the logs..."
+    You can access the logs for the AI node even when you aren't using the chat interface. Open up the **AI Agent** node and click on the **Logs** tab in the right hand panel.
+	![screenshot showing the Logs tab in the AIAgent](/_images/advanced-ai/ai-intro-logs.png)
+
+## 7. Changing the prompt
+
+The logs in the previous step reveal some extra data - the system prompt. This is the default message that the **AI Agent** primes the chat model with. From the log you can see this is set to "You are a helpful assistant". We can however change this prompt to alter the behavior of the chat model.
+
+1. Open the **AI Agent** node. In the bottom of the panel is a section labeled 'Options' and a selector labeled 'Add Option'. Use this to select 'System message'
+
+1. The system message is now displayed. This is the same priming prompt we noticed before in the logs. Change the prompt to something else to prime the chat model in a different way. You could try something like "You are a brilliant poet who always replies in rhyming couplets" for example.
+
+1. Close the node and return to the chat window. Repeat your message and notice how the output has changed.
+
+![image showing changed chat text, now in rhyming couplets](/_images/advanced-ai/ai-intro-poet.png)
+
+## 8. Persistence
+
+The chat model is now giving us useful output, but there is something wrong with it which will become apparent when you try to have a conversation.
+
+1. Use the chat and tell the chat model your name, for example "Hi there, my name is Nick".
+
+1. Wait for the response, then type the message "What's my name?". The AI will not be able to tell you, however apologetic it may seem. The reason for this is we are not saving the context. The AI Agent has no memory.
+
+![image showing a conversation illustrating the above](/_images/advanced-ai/ai-intro-memory.png)
+    
+1. In order to remember what has happened in the conversation, the AI Agent needs to preserve context. We can do this by adding memory to the **AI Agent** node. On the canvas click on the <span class="inline-image">![Add node icon](/_images/try-it-out/add-node-small.png){.off-glb}</span> on the bottom of the **AI Agent** node labeled "Memory".
```

Comment:
I just wanted to call out that this restarts the ordered list numbering. I think that's appropriate given that the previous one was a demo of the current behavior and now we're adding the persistence.

That being said, it might be good to break the two lists up with a sentence or two. Maybe just extract the first two sentences from this first step and put them before the list instead.

Reply:
I indented the image so now the numbers work and i think it provides a natural pause between the items